homework/hw1/task1/task1/bubbleSort.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace task1
+{
+    public static class BubbleSort
+    {
+        // Sorting int array with bubble sort.
+        public static IEnumerable<int> Sort(int[] sortedArray)
```

Comment:
```suggestion
        public static IEnumerable<int> Sort(this int[] sortedArray)
```
Можно реализовать сортировку в виде расширения для класса массива. Тогда сортировку можно будет вызвать по ООПшному: `array.Sort()`

Просто комментарий